test/Microsoft.DotNet.Tools.Publish.Tests/project.json

```diff
@@ -0,0 +1,24 @@
+﻿{
+  "version": "1.0.0-*",  
+  
+  "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.1-*",
+    "Microsoft.NETCore.Runtime": "1.0.1-*",
+    "Microsoft.NETCore.TestHost" : "1.0.0-*",
+    "System.Console": "4.0.0-beta-*",
+    "System.Runtime": "4.0.21-beta-*",
+    "System.AppContext": "4.0.1-beta-*",
```

Comment:
Should be able to upgrade this to use only `NETStandard.Library` and `Microsoft.NETCore.TestHost`

Crossgen bug shouldn't affect this.